diff --git a/node_modules/next-contentlayer/dist/.DS_Store b/node_modules/next-contentlayer/dist/.DS_Store
new file mode 100644
index 0000000..76d06dc
Binary files /dev/null and b/node_modules/next-contentlayer/dist/.DS_Store differ
diff --git a/node_modules/next-contentlayer/dist/hooks/useLiveReload.js b/node_modules/next-contentlayer/dist/hooks/useLiveReload.js
index a5dfff7..0a00a29 100644
--- a/node_modules/next-contentlayer/dist/hooks/useLiveReload.js
+++ b/node_modules/next-contentlayer/dist/hooks/useLiveReload.js
@@ -1,5 +1,7 @@
-import { addMessageListener } from 'next/dist/client/dev/error-overlay/websocket.js';
-import { useRouter } from 'next/router.js';
+import websocket from 'next/dist/client/dev/error-overlay/websocket.js';
+const { addMessageListener } = websocket;
+import router from 'next/router.js';
+const { useRouter } = router;
 // NOTE use ESM/CommonJS compat import here until resolved: https://github.com/facebook/react/issues/11503
 import React from 'react';
 /**
